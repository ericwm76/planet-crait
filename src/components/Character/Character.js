import React from 'react';

const Character = (props) => {
  return (
      <section className='character'>
        <h3>Name</h3>
        <p>Homeworld</p>
        <p>Population of Homeworld</p>
        <p>Species</p>
        <div className='related-films'>
          <p>Related Films</p>

        </div>
        <div className='favourite-button'>
        <i className="fal fa-jedi"></i>
        </div>
      </section>
  )
}

export default Character;