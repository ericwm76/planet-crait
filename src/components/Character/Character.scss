$crait-color : #c0392b;
$background-color : #faebd745;
 
.fa-jedi {
  color: black;
}

.character {
  border: 2px solid $crait-color;
  border-radius: 10px;
  background: $background-color;
  display: inline-block;
  height: 350px;
  margin: 10px;
  padding: 10px;
  width: 250px;
  &:hover {
    background-color: #EAFCEB; 
    box-shadow: 
      0 0 20px 5px #EAFCEB,  /* inner white */
      0 0 35px 15px #337F23, /* middle magenta */
      0 0 55px 25px #69A267; /* outer cyan */
    border: none;
  }
 }
