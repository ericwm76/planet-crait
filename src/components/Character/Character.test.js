import React from 'react';
import { shallow } from 'enzyme';
import Character from './Character';

describe('Character', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Character />)
  })

  it('should match snapshot with all data passed in correctly', () => {
      expect(wrapper).toMatchSnapshot();
  })
})