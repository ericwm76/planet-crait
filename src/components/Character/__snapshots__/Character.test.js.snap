// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character should match snapshot with all data passed in correctly 1`] = `undefined`;
